### ISSUE TITLE
Create a generic transaction to burn NFTs

### ISSUE THREAD

**Issue by joshuahannan** [onflow/flow-nft#236]
### Issue To Be Solved
Users need an easy way to burn NFTs from anywhere that doesn't require the specific app to implement burning functionality. A burner address like in Ethereum would be easiest, but that isn't really possible since you can't transfer to an uninitialized account.

### Suggest A Solution
* Maybe use a transaction that sends it to [BlackHole](https://github.com/fixes-world/token-list/blob/main/cadence/contracts/BlackHole.cdc)
* Maybe use a transaction that just sends it to `Burner`.
* Either way, it will likely be expected to just be a part of a web front end that anyone can sign in to with an FCL wallet.
* Provide a contract name, address, and NFT ID as arguments.

**Comment by austinkline** [comment]
You should be sending an nft to the Burner contract in case it has any callback functionality defined

**Comment by joshuahannan** [comment]
Of course, the `Burner` contract would always be the final destination no matter what. Some projects want an intermediate place for NFTs to go before being burnt though in order to have tracking for which NFTs are burnt and in which groups, etc. Still figuring out what that might look like

**Comment by btspoony** [comment]
That makes sense to me.
The main demand primarily comes from the Topshot Community.
But for the transaction, ideally it can accept multiple NFTs.

**Comment by joshuahannan** [comment]
Good point, I'll update it to accept multiple NFTs
